src/plugins/home/public/services/environment/environment.test.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EnvironmentService } from './environment';
+
+describe('EnvironmentService', () => {
+  describe('setup', () => {
+    test('allows multiple update calls', () => {
+      const setup = new EnvironmentService().setup();
+      setup.update({ ml: true });
+      setup.update({ apmUi: true });
```

Comment:
There's an `expect` missing in this test, isn't it?

Reply:
Sort of, the test is that this doesn't throw. I added an explicit `expect(...).not.toThrow()`